### DISCUSSION TITLE
React is not defined on new templates, and on all of my previous working templates?

### DISCUSSION THREAD

**Discussion by dmm22** [codesandbox/codesandbox-client#5414] [General] [answered]
getting "ReferenceError React is not defined" right after creating a new sandbox. None of my old stuff works either. What's up with that?

**Comment by siddharthkp** [comment, marked as answer]
> getting "ReferenceError React is not defined" right after creating a new sandbox.

Sorry about that, should be good now!

> None of my old stuff works either.

Can you pass me a sandbox link to debug?
